data/human/events.txt

```diff
@@ -2951,3 +2954,20 @@ mission "Geminus Rebuilt"
 	on offer
 		event "geminus rebuilt" 90
 		fail
+
+event "martini rebuilt"
+	planet "Martini"
+		description `Martini is a world of exotic beaches and perfect weather. The Republic's central bank and the galactic stock exchange were previously headquartered here, before a terrorist attack destroyed them and a large portion of the old capital city as well.`
+		description `	The capital; which will be rebuilt when the radiation clears away; is currently sealed off from public access, an unnecessary reminder of the day when millions of people died at the hands of Syndicate executives so that a recession could be averted. It is strange to hear the 'evil of money' now espoused by a population that once lived and thrived off that very object.`
+		description `	The new capital city is on the other side of the continent where the radiation from the nuclear bombings will not affect the rest of the population. It is not so grand as the old capital, but its population has swelled from resettlement from the irradiated areas. The inhabitants, many of whom are survivors of the blast, have a sort of 'will-do' attitude that only occurs after such a devastating and destructive event like an atomic bombing.`
```

Comment:
The last line feels weirdly here, as if <first> <last> knows first hand that there wouldn't be a higher depression rate after being nuked, or other.

Reply:
Disasters tend to create national unity. Think of events like 9-11 or the Christchurch shootings, instead of feeling more depressed people decided to come together and help each other. This is also after the civil war, and optimism in rebuilding their world would exist.

Reply:
Hmm, fair. Though it may be better to emphasise more on it.